assets/javascripts/kitten/components/form/checkbox.test.js

```diff
@@ -33,9 +34,11 @@ describe('Checkbox with default props', () => {
 
     it('renders input with passed props', () => {
       const input = component.find('input')
+      const errorClass = input.hasClass('is-error')
```

Comment:
Mieux, je pense qu'on peut éviter de devoir créer une const, en faisant quelque chose comme `expect(input).to.haveClass('is-error')`.